examples/benchmarking/M5-forecasting.ipynb

```diff
@@ -0,0 +1,3944 @@
+{
```

Comment:
<p>Before you used directly the benchmarking. Now you are using the RandomForestRegressor directly. Would it be better to first show how to make predictions and afterwards showing how benchmarking works? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime/pull/7113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>